code/main/edge/edgeRequestWorker.brs

```diff
@@ -153,9 +196,20 @@ function _adb_EdgeRequestWorker() as object
             ''' Add eventData under events key as an array
             requestBody.events = [eventData]
 
+            if not _adb_isEmptyOrInvalidMap(meta)
+                requestBody.meta = meta
+            end if
+
             return requestBody
         end function,
 
+        _getSdkConfigPayload: function(originalDatastreamId as string) as object
+            sdkConfig = {}
+            sdkConfig.datastream = {}
+            sdkConfig.datastream.original = originalDatastreamId
+        return sdkConfig
```

Comment:
Can we adjust the code format?

Reply:
Are you thinking of something like this:

```
sdkConfig = {"datastream" : { "original": originalDatastreamId } } 
```